### PR TITLE
add SmtpCheckTemplate

### PR DESCRIPTION
**Pull request by vorobevamv**

### CODE REVIEW

File: src/TextTemplateService.Data.Provider.MsSql.Ef/Migrations/20220707231500_AddTemplateTexts.cs [hunk 1]
```diff
@@ -157,6 +159,30 @@ protected override void Up(MigrationBuilder migrationBuilder)
...
             true
+          },
+          {
+            Guid.NewGuid(),
+            SmtpCheckTemplate,
+            "SMTP settings check",
+            "SMTP settings check",
+            "Hello!\n"
+            + "This email address was specified as the administrator's email address to receive the SMTP check's message.\n"
+            + "If you havn't submitted a request, ignore this message, please.\n"
```
Comment (VLiamin) [inline src/TextTemplateService.Data.Provider.MsSql.Ef/Migrations/20220707231500_AddTemplateTexts.cs]: it is better haven't
https://www.quora.com/Is-there-a-difference-between-havent-and-havnt

**Review by VLiamin — APPROVED** [review]

File: src/TextTemplateService.Data.Provider.MsSql.Ef/Migrations/20220707231500_AddTemplateTexts.cs (hunk 1 above)
Comment (vorobevamv) [reply src/TextTemplateService.Data.Provider.MsSql.Ef/Migrations/20220707231500_AddTemplateTexts.cs]: fixed)

File: src/TextTemplateService.Business/Commands/Template/EditTemplateCommand.cs [hunk 2]
```diff
@@ -13,6 +13,7 @@
 using LT.DigitalOffice.TextTemplateService.Mappers.Patch.Interfaces;
 using LT.DigitalOffice.TextTemplateService.Models.Dto.Requests.Template;
 using LT.DigitalOffice.TextTemplateService.Validation.Validators.Template.Interfaces;
+using Microsoft.AspNetCore.Http;
```
Comment (ndoriomedov) [inline src/TextTemplateService.Business/Commands/Template/EditTemplateCommand.cs]: do u use it?

**Review by ndoriomedov — APPROVED** [review]

File: src/TextTemplateService.Data.Provider.MsSql.Ef/Migrations/20220707231500_AddTemplateTexts.cs [hunk 3]
```diff
@@ -157,6 +159,30 @@ protected override void Up(MigrationBuilder migrationBuilder)
...
             true
+          },
+          {
+            Guid.NewGuid(),
+            SmtpCheckTemplate,
+            "SMTP settings check",
+            "SMTP settings check",
+            "Hello!\n"
+            + "This email address was specified as the administrator's email address to receive the SMTP check's message.\n"
+            + "If you haven't submitted a request, ignore this message, please.\n"
```
Comment (ndoriomedov) [inline src/TextTemplateService.Data.Provider.MsSql.Ef/Migrations/20220707231500_AddTemplateTexts.cs]: maybe "if you haven't submitted a request, please ignore this message."?

File: src/TextTemplateService.Business/Commands/Template/EditTemplateCommand.cs (hunk 2 above)
Comment (vorobevamv) [reply src/TextTemplateService.Business/Commands/Template/EditTemplateCommand.cs]: deleted

File: src/TextTemplateService.Data.Provider.MsSql.Ef/Migrations/20220707231500_AddTemplateTexts.cs (hunk 3 above)
Comment (vorobevamv) [reply src/TextTemplateService.Data.Provider.MsSql.Ef/Migrations/20220707231500_AddTemplateTexts.cs]: OK)

**Review by MariyaSin — APPROVED** [review]

**Pull request merged**